### PR TITLE
[zh-cn] Update job.md

### PR DESCRIPTION
**Pull request by Eason1118**
Keep it consistent with the original text and optimize the Chinese description

Original link: [https://github.com/kubernetes/website/pull/48378](https://github.com/kubernetes/website/pull/48378)

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 1]
```diff
@@ -1227,13 +1227,13 @@ Pod 终结器，Job 控制器就会给 Job 添加 `Complete` 或 `Failed` 终止
...
-`JobManagedBy` or the `JobPodReplacementPolicy` (enabled by default)
+_after_ all of the Pods are terminated. You can control this behavior by using the
+`JobManagedBy` and the `JobPodReplacementPolicy` (both enabled by default)
 [feature gates](/docs/reference/command-line-tools-reference/feature-gates/).
 -->
-在 Kubernetes v1.31 及更高版本中，控制器仅在所有 Pod 终止后添加 Job 终止状况。
-你可以使用 `JobManagedBy` 或 `JobPodReplacementPolicy`（默认启用）
-启用此行为的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
+在 Kubernetes v1.31 及更高版本中，控制器仅在 所有 Pod 都终止后 才会添加作业（Job）的终止条件。
+您可以通过使用 `JobManagedBy` 和 `JobPodReplacementPolicy`（都默认启用）
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1235]: No extra space between chinese characters;
avoid "您", use "你".
If English text is marking some text as italics, we change it to bold fonts.
Italic is not for Chinese characters.

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 2]
```diff
@@ -1227,13 +1227,13 @@ Pod 终结器，Job 控制器就会给 Job 添加 `Complete` 或 `Failed` 终止
...
+控制此行为的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1236]: 更改之后句子不通顺，谓语和宾语不搭配。

```suggestion
[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
```

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 3]
```diff
@@ -1231,9 +1231,9 @@ _after_ all of the Pods are terminated. You can control this behavior by using t
 `JobManagedBy` and the `JobPodReplacementPolicy` (both enabled by default)
 [feature gates](/docs/reference/command-line-tools-reference/feature-gates/).
 -->
-在 Kubernetes v1.31 及更高版本中，控制器仅在 所有 Pod 都终止后 才会添加作业（Job）的终止条件。
-您可以通过使用 `JobManagedBy` 和 `JobPodReplacementPolicy`（都默认启用）
-控制此行为的[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
+在 Kubernetes v1.31 及更高版本中，控制器仅在所有Pod都终止后才会添加作业（Job）的终止条件。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1234]: 中英文之间的空白要保留。英文的强调改成粗体。

```suggestion
在 Kubernetes v1.31 及更高版本中，控制器仅在所有 Pod 都终止**之后**才会添加作业（Job）的终止条件。
```

File: content/zh-cn/docs/concepts/workloads/controllers/job.md [hunk 4]
```diff
@@ -1231,9 +1231,9 @@ _after_ all of the Pods are terminated. You can control this behavior by using t
...
+你可以通过使用 `JobManagedBy` 和 `JobPodReplacementPolicy`（都默认启用）
+来控制此行为[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/)。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/workloads/controllers/job.md:1236]: 仍然不对，你控制的不是特性门控，你控制的是行为，方式是调整特性门控。

**Comment by Eason1118** [comment]
Thank you for your patient guidance, I will pay more attention next time

**Comment by tengqm** [comment]
Please squash your PR into a single commit.

**Comment by windsonsea** [comment]
@Eason1118 To read [中文本地化样式指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/) will be helpful to you.

**Pull request closed without merging**